packages/@aws-cdk/assert/lib/synth-utils.ts

```diff
@@ -7,7 +9,9 @@ export class SynthUtils {
   }
 
   public static templateForStackName(session: ISynthesisSession, stackName: string): any {
-    return session.store.readJson(session.getArtifact(stackName).properties!.templateFile);
+    const fileName = session.getArtifact(stackName).properties!.templateFile;
+    const filePath = path.join(session.outdir, fileName);
```

Comment:
Couldn't the `session` (who has all the pieces here) return the absolute path directly?

Reply:
Session is now `CloudAssembly` yey!